plugins.d/System_Settings/Secupdates_adv_conf.py

```diff
@@ -39,23 +39,23 @@ def conf_alternate() -> None:
     new_link(FILE_PATH, CONF_ALT)
 
 
-def check_paths() -> tuple[int, list[str] | str]:
+def check_paths() -> tuple[int, list[str]]:
     errors: list[str] = []
     for _path in [FILE_PATH, CONF_DEFAULT, CONF_ALT]:
         if not exists(_path):
-            errors.append('Path not found: {}'.format(str(_path)))
+            errors.append(f'Path not found: {_path}')
     if errors:
         return 2, errors
     if islink(FILE_PATH):
-        _target_path = FILE_PATH.resolve()
+        _target_path = os.readlink(FILE_PATH)
         if _target_path == CONF_DEFAULT:
-            return 0, 'default'
+            return 0, ['default']
```

Comment:
This change will break line 104, you'll need to either change that line, or this line (line 104 checks against the string `"default"` rather than the first item of a list)

Reply:
Great catch, thanks! Although there is another issue...